Exception java.lang.NoClassDefFoundError: com/google/zxing/NotFoundException

Hallo, ich bekomme beim durchführen von sh parsepdfs.sh folgende Fehlermeldung. Was kann ich tun?

> sh parsepdfs.sh        
>       
> Exception in thread "main" java.lang.NoClassDefFoundError: com/google/zxing/NotFoundException
> 	at com.google.zxing.client.j2se.CommandLineRunner.main(CommandLineRunner.java:99)
> 
> Caused by: java.lang.ClassNotFoundException: com.google.zxing.NotFoundException
> 	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:606)
> 	at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:168)
> 	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
> 	... 1 more

--- a/parsepdfs.sh
+++ b/parsepdfs.sh
@@ -4,7 +4,7 @@ SCRIPTDIR=$(dirname $0)
 # Download the binaries (if not yet existing)
 VERSION=3.3.1
 for lib in core javase; do
-  [ -e $SCRIPTDIR/zx-$lib.jar ] || wget http://repo1.maven.org/maven2/com/google/zxing/$lib/$VERSION/$lib-$VERSION.jar -O $SCRIPTDIR/zx-$lib.jar
+  [ -e $SCRIPTDIR/zx-$lib.jar ] || wget https://repo1.maven.org/maven2/com/google/zxing/$lib/$VERSION/$lib-$VERSION.jar -O $SCRIPTDIR/zx-$lib.jar
 done
 [ -e $SCRIPTDIR/jcommander.jar ] || wget https://repo1.maven.org/maven2/com/beust/jcommander/1.72/jcommander-1.72.jar -O $SCRIPTDIR/jcommander.jar
 

